pkg/integrations/prometheus/utils/operatorutils.go

```
@@ -180,7 +180,7 @@ func SetFinalResult(queryOp *structs.QueryArithmetic, finalResult map[string]map
 			}
 		}
 	default:
-		log.Errorf("SetFinalResult: does not support using this operator: %v", queryOp.Operation)
+		log.Debugf("SetFinalResult: does not support using this operator: %v", queryOp.Operation)
 	}
```

Why do we want to silently fail? I'd think we'd want to return an error, and log the error somewhere

same comment line 196

---

Hmm this error was flooding the logs. Actually I will revert this change. And will fix this in the PR of the above mentioned TODO.